packages/web/components/input/search-box.tsx

```diff
@@ -20,9 +20,10 @@ const searchBoxClasses = cva(
        * - letter spacing
        */
       size: {
-        small: "h-10 px-5 w-max [&_input]:text-body2 [&_input]:font-body2",
-        medium: "h-12 px-5 w-max [&_input]:text-body2 [&_input]:font-body2",
-        large: "h-14 px-5 w-max [&_input]:text-body1 [&_input]:font-body2",
+        small: "h-10 pl-5 pr-1 w-max [&_input]:text-body2 [&_input]:font-body2",
```

Comment:
It's very nitpicky: but it seems now the search box changes size when it's focused now, causing neighboring elements to shift. Don't let it block you, but if we could avoid that it'd be idea.

Reply:
I wasn't able to find a quick fix for this, I thought `box-sizing: border-box;` would work but that's not the case. Will need more time to fix it.

Reply:
Fixed by making border width consistent across states (default, onFocus).